DataFormats/Scouting/interface/Run3ScoutingPFJet.h

```diff
@@ -0,0 +1,136 @@
+#ifndef DataFormats_Run3ScoutingPFJet_h
+#define DataFormats_Run3ScoutingPFJet_h
+
+#include <vector>
+
+//class for holding PF jet information, for use in data scouting
+//IMPORTANT: the content of this class should be changed only in backwards compatible ways!
+class Run3ScoutingPFJet {
+public:
+  //constructor with values for all data fields
+  Run3ScoutingPFJet(float pt,
+                    float eta,
+                    float phi,
+                    float m,
+                    float jetArea,
+                    float chargedHadronEnergy,
+                    float neutralHadronEnergy,
+                    float photonEnergy,
+                    float electronEnergy,
+                    float muonEnergy,
+                    float HFHadronEnergy,
+                    float HFEMEnergy,
+                    int chargedHadronMultiplicity,
+                    int neutralHadronMultiplicity,
+                    int photonMultiplicity,
+                    int electronMultiplicity,
+                    int muonMultiplicity,
+                    int HFHadronMultiplicity,
+                    int HFEMMultiplicity,
+                    float HOEnergy,
+                    float csv,
+                    float mvaDiscriminator,
+                    std::vector<int> constituents)
+      : pt_(pt),
+        eta_(eta),
+        phi_(phi),
+        m_(m),
+        jetArea_(jetArea),
+        chargedHadronEnergy_(chargedHadronEnergy),
+        neutralHadronEnergy_(neutralHadronEnergy),
+        photonEnergy_(photonEnergy),
+        electronEnergy_(electronEnergy),
+        muonEnergy_(muonEnergy),
+        HFHadronEnergy_(HFHadronEnergy),
+        HFEMEnergy_(HFEMEnergy),
+        chargedHadronMultiplicity_(chargedHadronMultiplicity),
+        neutralHadronMultiplicity_(neutralHadronMultiplicity),
+        photonMultiplicity_(photonMultiplicity),
+        electronMultiplicity_(electronMultiplicity),
+        muonMultiplicity_(muonMultiplicity),
+        HFHadronMultiplicity_(HFHadronMultiplicity),
+        HFEMMultiplicity_(HFEMMultiplicity),
+        HOEnergy_(HOEnergy),
+        csv_(csv),
+        mvaDiscriminator_(mvaDiscriminator),
+        constituents_(constituents) {}
+  //default constructor
+  Run3ScoutingPFJet()
+      : pt_(0),
+        eta_(0),
+        phi_(0),
+        m_(0),
+        jetArea_(0),
+        chargedHadronEnergy_(0),
+        neutralHadronEnergy_(0),
+        photonEnergy_(0),
+        electronEnergy_(0),
+        muonEnergy_(0),
+        HFHadronEnergy_(0),
+        HFEMEnergy_(0),
+        chargedHadronMultiplicity_(0),
+        neutralHadronMultiplicity_(0),
+        photonMultiplicity_(0),
+        electronMultiplicity_(0),
+        muonMultiplicity_(0),
+        HFHadronMultiplicity_(0),
+        HFEMMultiplicity_(0),
+        HOEnergy_(0),
+        csv_(0),
+        mvaDiscriminator_(0),
+        constituents_(std::vector<int>()) {}
```

Comment:
Implicit call to default constructor is sufficient
```suggestion
        mvaDiscriminator_(0) {}
```